src/main/java/catan/domain/model/game/Resources.java

```diff
@@ -75,6 +76,43 @@ public void setStone(int stone) {
         this.stone = stone;
     }
 
+    public Integer findResource(HexType resource) {
```

Comment:
we count number of resources that player has, not resource itself, this name does not fit this case